invenio_jobs/services/services.py

```diff
@@ -263,10 +262,6 @@ def create(self, identity, job_id, data, uow=None):
                 queue=run.queue,
             )
         )
-        # Make sure this is the last operation in the unit of work
-        # so tht the post_commit (that is resetting the job context)
-        # is executed after the TaskOp post_commit.
-        uow.register(JobContextOp({"run_id": str(run.id), "job_id": str(job_id)}))
```

Comment:
Removing this will cause that any run triggered manually and not scheduled will not log anything, so better to not remove it 😉